src/containers/AdminCampaignEdit/sections/CampaignContactsForm/components/ConfigureColumnMappingDialog.tsx

```diff
@@ -141,15 +141,18 @@ const ConfigureColumnMappingDialog: React.FC<ConfigureColumnMappingDialogProps>
     }
 
     const remappedColumns = mappedColumns.map((column) => {
-      const alias = fieldAliases[column.remap];
-      return {
-        column: column.column,
-        remap: alias ?? column.remap
-      };
+      if (column.remap) {
+        const alias = (fieldAliases as any)[column.remap];
+        return {
+          column: column.column,
+          remap: alias ?? column.remap
+        };
+      }
+      return null;
     });
 
     setError(null);
-    onSave(remappedColumns);
+    onSave(remappedColumns as ColumnMapping[]);
```

Comment:
do you need to filter out null or something? ColumnMapping as a type does not allow for null

Reply:
yeah i think this was me taking too many shortcuts in adding types earlier in the file making it even harder later 😭